src/ProfileTracer.cpp

```diff
@@ -78,6 +78,10 @@ namespace geopm
                 case EVENT_EPOCH_COUNT:
                     result = string_format_integer(value);
                     break;
+                case EVENT_SHORT_REGION:
+                    // integer value of number of short regions
+                    result = string_format_integer(value);
```

Comment:
```suggestion
                    result = string_format_hex(value);
```
This should provide the hex value of region as it is written.  
We may want to add an EVENT_SHORT_REGION count (which would use the string_format_integer), but for #1766 only the region addition is needed.